app/models/site_config.rb

```diff
@@ -4,14 +4,15 @@
 class SiteConfig < RailsSettings::Base
   self.table_name = "site_configs"
 
-  validates :var, presence: true
```

Comment:
This has me stumped a bit. While this line works perfectly fine on `master` it breaks on this branch, even though the gem version is the same and no relevant code was changed.

Master:
![image](https://user-images.githubusercontent.com/47985/105675655-fe4ac200-5f1b-11eb-8aa9-57ab20caa28e.png)

This branch:
![image](https://user-images.githubusercontent.com/47985/105675682-09055700-5f1c-11eb-9bf1-ea366659398f.png)

The column itself already has a non-null constraint, so it's not possible to insert an invalid value. We also never directly interact with `SiteConfig` records but only do so through the gem's interface. 

I disabled this in order to not get stuck in a debugging rabbit hole for a validation that only seems marginally useful to me, but I'm happy to dig deeper if people feel strongly about it.

Reply:
For the record, @rhymes and I discussed this on Slack and agreed that this validation can be safely removed.